### ISSUE TITLE
[accessibility]: checkbox + radio are missing role

### ISSUE THREAD

**Issue by rachelbt** [Vonage/vivid-3#855]
role="radio" 
role="radiogroup"
role="checkbox"

**Comment by yinonov** [comment]
<img width="545" alt="image" src="https://user-images.githubusercontent.com/15140652/203325840-5d30a3e5-1807-4227-8dcb-93db383a659d.png">
??
